### ISSUE TITLE
Installing Beta on Firefox

### ISSUE THREAD

**Issue by wisdomlight** [tridactyl/tridactyl#2978]
I just came across tridactyl
the GitHub page said installing for Firefox  (on latest Debian Bullseye) is just a click.
So I clicked and a downloaded .xpi file started.
I extracted the file. But now what???

**Comment by bovine3dom** [comment]
That's interesting, it "just works" for me.

Don't extract the file - try opening the xpi with Firefox or clicking and dragging the file onto Firefox. Otherwise you can go to about:addons and click "install add-on from file" somewhere (I think they've hidden it a little bit so you might have to dig around).

**Comment by wisdomlight** [comment]
ok i installed the normal one from the firefox which at the moment seems to be more than what i can chew...

**Comment by wisdomlight** [comment]
Thanks It seems to have worked. 
I opend it with Firefox and it installed 
cheers

**Comment by bovine3dom** [comment]
If you installed the version from the Firefox site too, make sure you disable one of them in about:addons or weird things will happen : )

**Comment by wisdomlight** [comment]
ok it all works 
thanks I really enjoy your addon
would be nice if the tutorial was a bit more welcoming for those of us who are not masters of the Vim universe.

**Comment by bovine3dom** [comment]
That would be nice - do you have any recommendations for places we could improve it?

**Comment by wisdomlight** [comment]
First and foremost you just won the cool-dudes-not-afraid-of-improving award. Which is amazing. 
Secondly yes the main point on improving is as always documentation. 
And it really depends on which audience are you aiming at. 
If you are hoping to convert people to using a keyboard based internet navigation then my immediate two suggestions are 
1. Glossary 
2. Examples of what the things mean. 


If you check qutebtowser the actually created a training dojo for their browser. But obviously that’s a great time investment you might not be able to do. 

And a third suggestion might be short videos of how things look on the screen when used live.

**Comment by bovine3dom** [comment]
Reopening this as someone else had trouble. We should probably document how to install it manually in the readme.

**Comment by xradiation** [comment]
Hi, 
Had the same problem just now, For some reason firefox is stripping the `.xpi` extension with `.zip` one ( For security reasons !!!!??? ). I tried to open the `.zip` file with firefox download box & from the terminal and both ways ended up coping the tridactyl file to `/tmp/mozila...` . 

I've found that the only way is to swap the extension to `.xpi` and only then it actually work as intended.

Best Regards.
